src/server/services/wms/qgswmsparameters.h

```diff
@@ -1309,6 +1311,22 @@ namespace QgsWms
        */
       QMap<QString, QString> dimensionValues() const;
 
+      /**
+       * \returns true if the FORCE_MTEXT parameter is set and the DXF should
+       * be produced with MTEXT instead of TEXT.
+       *
+       * \since QGIS 3.12
+       */
+      bool noMText() const;
+
+      /**
+       * \returns true if the FORCE_2D parameter is set and the DXF should
+       * be produced in 2D.
+       *
+       * \since QGIS 3.12
+       */
+      bool force2D() const;
```

Comment:
isForce2D? is2D?

Reply:
What's wrong with the current one?

Reply:
Ah, it could be misread as an imperative command instead of a getter?

Reply:
Yes :)